Formula/darktable.rb

```diff
@@ -45,38 +50,23 @@ def install
 
   def test_convert_to(extension)
     expected_file = "image.#{extension}"
-    ohai "Converting raw to #{extension}"
-    system bin/"darktable-cli", "image.dng", "-o", expected_file
+    system bin/"darktable-cli", "homebrew.raw", "-o", expected_file
     assert File.size?(expected_file)
   end
 
-  def test_lua
-    require "open3"
+  test do
+    testpath.install resource("raw_sample")
+    test_convert_to("jpg")
+    test_convert_to("tif")
+    test_convert_to("png")
```

Comment:
Can’t you use a block in a loop like `%w[jpg tif png].each do |extension|` instead of having a separate function?